lighthouse-core/computed/metrics/timing-summary.js

```diff
@@ -109,7 +115,10 @@ class TimingSummary {
       observedSpeedIndexTs: (speedline.speedIndex + speedline.beginning) * 1000,
     };
     /** @type {Record<string,boolean>} */
-    const debugInfo = {lcpInvalidated: traceOfTab.lcpInvalidated};
+    const debugInfo = {
+      lcpInvalidated: traceOfTab.lcpInvalidated,
+      lcpAllFramesInvalidated: traceOfTab.lcpAllFramesInvalidated,
```

Comment:
with invalidations already gone in canary, I don't know if this is worth tracking 🤷

Reply:
Remove both `lcpInvalidated` and `lcpAllFramesInvalidated`?

Reply:
might be a smidge early to remove `lcpInvalidated`, I was just making the case for why we don't need to track `lcpAllFramesInvalidated`